stdlib/multiprocessing/heap.pyi

```diff
@@ -0,0 +1,32 @@
+import sys
+from _typeshed import Incomplete
+from collections.abc import Callable
+from mmap import mmap
+from typing_extensions import TypeAlias
+
+_Block: TypeAlias = tuple[Arena, int, int]
```

Comment:
Nit: can we move this type alias declaration below the `Arena` class definition? Type checkers support forward references in stub files, but it's making me do a double take every time I see this :)

Reply:
Sure, no worries. I think of it more in terms of stub organization (TypeAliases first, then implementation using them), but not a big deal at all, especially for short stubs like this

Reply:
> Sure, no worries. I think of it more in terms of stub organization (TypeAliases first, then implementation using them), but not a big deal at all, especially for short stubs like this

In general, yes, but it's nice to avoid forward references where possible :)